packages/enzyme-adapter-react-17/package.json

```diff
@@ -0,0 +1,73 @@
+{
+  "name": "enzyme-adapter-react-17",
+  "version": "1.15.3",
+  "description": "JavaScript Testing utilities for React",
+  "homepage": "https://enzymejs.github.io/enzyme/",
+  "main": "build",
+  "scripts": {
+    "clean": "rimraf build",
+    "lint": "eslint --ext js,jsx .",
+    "pretest": "npm run lint",
+    "prebuild": "npm run clean",
+    "build": "babel --source-maps=both src --out-dir build",
+    "watch": "npm run build -- -w",
+    "prepublish": "not-in-publish || (npm run build && safe-publish-latest && cp ../../{LICENSE,README}.md ./)"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/enzymejs/enzyme.git",
+    "directory": "packages/enzyme-adapter-react-17"
+  },
+  "keywords": [
+    "javascript",
+    "shallow rendering",
+    "shallowRender",
+    "test",
+    "reactjs",
+    "react",
+    "flux",
+    "testing",
+    "test utils",
+    "assertion helpers",
+    "tdd",
+    "mocha"
+  ],
+  "author": "Jordan Harband <ljharb@gmail.com>",
+  "funding": {
+    "url": "https://github.com/sponsors/ljharb"
+  },
+  "license": "MIT",
+  "dependencies": {
+    "enzyme-adapter-utils": "^1.13.1",
+    "enzyme-shallow-equal": "^1.0.4",
+    "has": "^1.0.3",
+    "object.assign": "^4.1.0",
+    "object.values": "^1.1.1",
+    "prop-types": "^15.7.2",
+    "react-is": "^17.0.0-rc.0",
+    "react-test-renderer": "^17.0.0-rc.0",
+    "semver": "^5.7.0"
+  },
+  "peerDependencies": {
+    "enzyme": "^3.0.0",
+    "react": "^17.0.0-rc.0",
+    "react-dom": "^17.0.0-rc.0"
```

Comment:
Not sure about versions there, suggestions?

Reply:
Perhaps `^17.0.0-rc.0 || ^17.0.0`?